src/main/java/uk/gov/hmcts/probate/model/ApplicationType.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.probate.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum ApplicationType {
```

Comment:
is there some meaningful way to incorporate the PrintTemplateApplicationType enum functionality here too?